docs/section4/4-3.md

```diff
@@ -0,0 +1,39 @@
+# Airside System – Rule 4-3
+
+**Rule ID:** 4-3
+ 
+**Rule Description:** Schedules for HVAC fans that provide outdoor air for ventilation shall run continuously whenever spaces are occupied.  
+**Rule Assertion:** P-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-4 Schedules for the proposed building excluding exception #1. 
+**Appendix G Section Reference:** None
+**Data Lookup:** None 
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem  
+
+**Applicability Checks:** 
+
+1. This rule does not apply when no heating and/or cooling system is to be installed, and a heating or cooling system is being simulated only to meet the requirements described in Section G3.1-10 HVAC Systems proposed column c and d.
+ 
+**Function Call:** 
+
+1. get_HVAC_systems_applicable_10cd
+
+**Manual Check:** None  
+
+**Rule Logic:**
+**Applicability Check 1:** 
+- Call function to get list of inapplicable HVAC systems: `inapplicable_HVAC_systems_list_p = get_HVAC_systems_applicable_10cd(U_RMR,P_RMR)`
+- For each HeatingVentilationAirAconditioningSystem in the P_RMR check if this check is applicable: `for each hvac_p in P_RMR...hvac_systems:`
+    - Check if hvac_p is not in the list of inapplicable systems, if it is not found then set applicability flag to true: `if hvac_p not in inapplicable_HVAC_systems_list_p: rule_applicability_flag = TRUE`   
```

Comment:
Maria to email PNNL or someone about how this should be handled.

Reply:
Christina, I emailed the question. I wonder if the current App G rule can even be implemented as written in the simulation tools, as it seems to require in some cases having two different HVAC system types serving the same zone. Consider a proposed design involving a 50,000 SF elementary school with unit ventilators (UVs) serving each classroom and no cooling specified. Based on G3.1 #10 (d), the proposed design should be modeled with the same cooling system type as in the baseline, which gives us System 5. So in this case, it seems that the proposed design should be modeled with UVs running continuously during heating season, and with cycling System 5 providing cooling but no OA during cooling season.

Reply:
@mkarpman in the scenario you've described above - there would need to be 2 HVAC systems in the RMR, right? So this should not be a problem. But in the scenario where the there is a single system with as design H and baseline C, it would have the same fan and be represented as one system in the schema. Correct?

Reply:
Correct - if a separate HVAC system is created, it's not a problem from the RMR structure perspective but not sure how it would actually be modeled, as there would be effictively two separate HVAC systems, one operating in cooling mode only and another in heating mode only. So during swing months, they would need to alternate based on weather?

If a single HVAC system is modeled for the proposed, then it becomes a problem.

Reply:
@mkarpman i assume in the UV + cycling system 5 - the rule would be flagged as fail since the UV will likely have min OA as 0. I think that is the right approach- anything I'm missing here?